todo-app/src/Stores/TasksListStore.js

```diff
@@ -0,0 +1,151 @@
+import {action,configure,observable} from 'mobx'
+import moment from 'moment'
+
+import {loadFromLocalStorage,saveToLocalStorage} from '../localStorage'
+import todoList from '../todoList.json'
+
+configure({
+    enforceActions: true
+})
+
+
+
+class TasksListStore {
+
+    @observable
+    todoList = loadFromLocalStorage() || todoList
+
+
+    @observable
+    selectedToEdit = null
+
+    @observable
+    editedTask = ''
+
+    @observable
+    newTask = {
+        description: '',
+        date: '',
+    }
+
+    @action
+    selected = (id) => {

```

Comment:
The naming here could be a bit better. It's _toggling_ the selection of todo items.